samples/BikeSharingApp/Bikes/TESTING.md

```diff
@@ -0,0 +1,8 @@
+### Local Debugging pre-requisites
+* ```npm install```
+* [Debugging node js app in VS Code](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
```

Comment:
>node js [](start = 13, length = 7)

Nit: "Node.js" to respect the format they use for their name?

Reply:
Updated